typescript/src/README.md

```diff
@@ -12,13 +13,21 @@
 - [Support](#support)
 - [Security](#security)
 
-> [!TIP]
-> If you're looking to contribute to the SDK, please see the [Contributing Guide](https://github.com/coinbase/cdp-sdk/blob/main/typescript/CONTRIBUTING.md).
+> [!TIP] If you're looking to contribute to the SDK, please see the [Contributing Guide](https://github.com/coinbase/cdp-sdk/blob/main/typescript/CONTRIBUTING.md).
```

Comment:
Can you revert this change? This renders as

<img width="653" alt="image" src="https://github.com/user-attachments/assets/dca3de57-25d2-4ba6-99ec-6196c91b9cfb" />

Instead of 

<img width="624" alt="image" src="https://github.com/user-attachments/assets/95b5621f-972b-4805-a051-b0cd6540492a" />

Maybe we should add this file to the .prettierignore...